include/network.php

```diff
@@ -487,6 +522,37 @@ function allowed_url($url) {
 }
 
 /**
+ * Checks if the provided url domain isn't on the domain blacklist.
+ * Return true if the check passed (not on the blacklist), false if not
+ * or malformed URL
+ *
+ * @param string $url The url to check the domain from
+ * @return boolean
+ */
+function check_domain_blocklist($url) {
```

Comment:
I hadn't called the function "check_domain_blacklist", because I like readable "if" statements. And a "If check domain blacklist then ..." doesn't sound like a good sentence :-)

I guess I had called it "domain_blacklisted" or "domain_allowed" so that the "if" would have sounded like: "if domain blacklisted then ...".